crates/sp-objects/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "sp-objects"
+version = "1.0.0"
```

Comment:
I think we use `0.1.0` by default everywhere at this moment :D. https://github.com/subspace/subspace/blob/6262ca1c9523667e717dce67408f28ccbe92d203/crates/pallet-grandpa-finality-verifier/Cargo.toml#L3 probably also needs to be updated to be `0.1.0`.

Reply:
hmm. why 0.1.0 🤔 ?